### ISSUE TITLE
Markerclusterer plus not working properly at low zoom levels

### ISSUE THREAD

**Issue by GoogleCodeExporter** [seancallinan/google-maps-utility-library-v3#130]
```
Demo link or sample code:
http://google-maps-utility-library-v3.googlecode.com/svn/tags/markerclustererplu
s/2.0.3/examples/advanced_example.html

What steps will reproduce the problem?
1. Go to demo page
2. Move the map over russia
3. Zoom out to zoom level 0
4. Observe number of clusters and locations(some missing)
5. Move the map horizontally by the width of the map
6. Observe new (actual) clusters

Expected result:
At zoom level 0 all clusters are loaded initially including all markers

Actual result:
At zoom level 0/1(or possibly other zoom levels which allow the map to be 
displayed multiple times horizontally) some clusters and markers are missing.

Version: 2.0.3

Browser / Operating System:
[FF3 / WIN XP]


*********************************************************
Tip: Star this issue (next to title) to receive notifications of status
changes against this issue, also used as a gauge for how many people are
interested in seeing it resolved.
*********************************************************

```

Original issue reported on code.google.com by `lutz.bie...@gmail.com` on 22 Sep 2011 at 11:27

**Comment by GoogleCodeExporter** [comment]
```
This problem occurs in both MarkerClustererPlus and MarkerClusterer.

It seems to be caused by (yet another) bug in the nightly build of 3.6. If you 
change the version number of the Google API to v=3.5 everything seems to work 
as expected.


```

Original comment by `garylitt...@gmail.com` on 23 Sep 2011 at 1:46
- Changed state: **Confirmed**

**Comment by GoogleCodeExporter** [comment]
```
Confirmed; fixing the url at v=3.5 fixed the issue. 


```

Original comment by `r...@flavor8.com` on 25 Sep 2011 at 8:32

**Comment by GoogleCodeExporter** [comment]
```
The bug is also reported here:

http://code.google.com/p/gmaps-api-issues/issues/detail?id=3651
```

Original comment by `garylitt...@gmail.com` on 26 Sep 2011 at 7:38

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the work around guys.
```

Original comment by `tim.poul...@gmail.com` on 28 Jan 2012 at 4:18

**Comment by GoogleCodeExporter** [comment]
```
Setting version to 3.5 won't work anymore since Google is forcing 3.6 at 
minimum.
```

Original comment by `pholm...@gmail.com` on 10 Feb 2012 at 7:17

**Comment by GoogleCodeExporter** [comment]
```
Version 3.5 is accessible in Ireland as of now (10 Feb 2012).
Is there a fix for this in later version of markclusterer plus.
At the moment we're using googlemaps 3.5 and markclustererplus 2.0.3 which 
works for now.
```

Original comment by `lutz.bie...@gmail.com` on 10 Feb 2012 at 10:04

**Comment by GoogleCodeExporter** [comment]
```
The problem will presumably disappear when Google fixes this bug:

http://code.google.com/p/gmaps-api-issues/issues/detail?id=3651

or

http://code.google.com/p/gmaps-api-issues/issues/detail?id=3914
```

Original comment by `garylitt...@gmail.com` on 12 Feb 2012 at 4:51

**Comment by GoogleCodeExporter** [comment]
```
Setting version to 3.5 don't work anymore in France since Google is forcing 3.6 
at minimum.

It's really annoying because I can not use MarkerClusterer ...
```

Original comment by `etienne....@gmail.com` on 16 Feb 2012 at 10:18

**Comment by GoogleCodeExporter** [comment]
```
The 3.5 fix is no longer working in Ireland either (16 Feb 2012)
```

Original comment by `eoghan.o...@gmail.com` on 16 Feb 2012 at 11:23

**Comment by GoogleCodeExporter** [comment]
```
There is a fix for this in the current trunk release (2.0.9).
```

Original comment by `garylitt...@gmail.com` on 21 Feb 2012 at 7:35
- Changed state: **FixedNotReleased**

**Comment by GoogleCodeExporter** [comment]
Original comment by `garylitt...@gmail.com` on 22 Feb 2012 at 5:31
- Changed state: **Fixed**
